### PR TITLE
Added flexbox demo

### LINKED ISSUE #36: Create demo for flexbox
Added a demo for the flexbox components

https://www.getpapercss.com/#flexbox

### PR DESCRIPTION
**Pull request by CalebBassham**
Added a demo for flexbox components, closes #36

### CODE REVIEW

File: demo/router.js [hunk 1]
```diff
@@ -52,6 +53,11 @@ export default new Router({
       path: '/component/popover',
       name: 'popover',
       component: Popover
+    },
+    {
+      path: '/flexbox',
```
Comment (TotomInc) [inline demo/router.js:58]: In the path you forgot `/component`.

**Review by TotomInc — CHANGES_REQUESTED** [review]

**Comment by TotomInc** [comment]
Flexbox should be a component IMO, what are your thoughts @DarkPaladinUHC?

**Comment by CalebBassham** [comment]
I think that it shouldn't be as it isn't a component that like alert or badge for instance that you can import. You can't import flexbox like all of the other components so I think that it would be less confusing. Like you wouldn't consider Typography a component. For a component I would do just col or row but they don't make much sense by themselves.

**Comment by TotomInc** [comment]
That's okay, this is why the path is only `/flexbox`.

**Review by TotomInc — APPROVED** [review]

**Pull request merged**